explorer/testdata/for/for-loop.carbon

```diff
@@ -0,0 +1,52 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// RUN: %{explorer} %s 2>&1 | \
+// RUN:   %{FileCheck} --match-full-lines --allow-unused-prefixes=false %s
+// RUN: %{explorer} --parser_debug --trace_file=- %s 2>&1 | \
+// RUN:   %{FileCheck} --match-full-lines --allow-unused-prefixes %s
+// AUTOUPDATE: %{explorer} %s
+// CHECK: result: 8
+
+package ExplorerTest api;
+
+
+
+
+fn Main() -> i32 {
+  var ar: [i32; 4] = (0, 1,2,3);
+  var i : i32 = __intrinsic_array_size(ar);
```

Comment:
We generally try to avoid using intrinsics in user code, and it looks like `i` isn't used anyway. Can we remove this line?

Reply:
removed